schema/trigger/moduleleveltrigger.jsonnet

```diff
@@ -27,6 +29,7 @@ local types = {
       s.field("td_out_of_timeout", self.td_out_of_timeout_b, doc="Option to drop TD if TC comes out of timeout window"),
       s.field("buffer_timeout", self.time_t, 100, doc="Buffering timeout [ms] for new TCs"),
       s.field("td_readout_limit", self.time_t, 1000, doc="Time limit [ms] for the length of TD readout window"),
+      s.field("ignore_tc", self.tc_types, [], doc="List of TC types to be ignored"),
```

Comment:
This is `ignore_tc`, while daqconf uses an `mlt_ignore_tc`.

Reply:
`ignore_tc` is linked to `mlt_ignore_tc` in daqconf, file python/daqconf/apps/trigger_gen.py of PR 259 as:
ignore_tc=MLT_IGNORE_TC,

this is not an issue and we were able to build and generate configs with this with no issues. What is the exact error you are seeing? Cheers.